localstripe/resources.py

```diff
@@ -2878,3 +2878,67 @@ def __init__(self, card=None, customer=None, **kwargs):
 
         self.type = 'card'
         self.card = card_obj
+
+
+class Balance(StripeObject):
+    object = 'balance'
+    # Balance does not have id prefix
+    _id_prefix = 'balance_'
```

Comment:
Then it shouldn't b defined ;-)

Reply:
Actually in the Payouts PR I found it it does, its `ba_`, adding it.